SampleViewer/SampleManager.h

```diff
@@ -128,39 +146,112 @@ class SampleManager : public QObject
   QVariantMap toVariantMap(const QString& json);
   QVariant fileUrl(const QString& scheme, const QString& path);
   QString readTextFile(const QString& filePath);
-  SampleListModel* samples() const { return m_allSamples; }
-  SampleListModel* featuredSamples() const { return m_featuredSamples; }
-  CategoryListModel* categories() { return m_categories; }
-  CurrentMode currentMode() { return m_currentMode; }
+
+  SampleListModel* samples() const
```

Comment:
I know these were like this before, but please consider moving their implementation to the .cpp files. That shouldn't be too much scope creep.